.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [ 1, {
+      "js": "always", "json": "always"
+    }]
+  },
+  "ignorePatterns": [
+    "dist/",
+    "build/"
+  ]
+}
```

Comment:
- [ **OPTIONAL** ] In `index.js` file:
![Screenshot from 2022-10-08 01-25-50](https://user-images.githubusercontent.com/83042742/194675117-b98fd991-ee5a-4354-a527-8613575ac971.png)
Kindly make sure that the user is still able to delete a unique task to match the expected functionalities. Right now the user can not delete a task he wants if he doesn't use the `Clear All completed` button and for that, it requires that the task is completed.

Reply:
Thank you for this comment! I fixed a linter in a way that broke this functionality, thank you for bringing it to my attention! It is now fixed, and delete button works! 😸